src/LibraryViewExtensionWebView2/LibraryViewExtensionWebView2.csproj

```diff
@@ -14,12 +14,38 @@
       <Deterministic>true</Deterministic>
     <EnableDefaultPageItems>false</EnableDefaultPageItems>
   </PropertyGroup>
+  
+  <Target Name="NpmRunBuild" BeforeTargets="BeforeBuild">
+        <!--This command updates the npm registry configuration if necessary-->
+        <Exec Command="$(PowerShellCommand) -ExecutionPolicy ByPass -Command $(SolutionDir)\setnpmreg.ps1" />
+        <!--This command gets a specific build of the notifcation center from npm-->
+        <Exec Command="npm pack @dynamods/librariejs@1.0.4" />
+    </Target>
+
+    <Target Name="ExtractTGZFile" BeforeTargets="BeforeBuild">
+        <!--Locates the .tgz files-->
+        <ItemGroup>
+            <TGZFiles Include="./dynamods-librariejs-*.tgz" />
+        </ItemGroup>
+
+        <!--Gets the last one-->
+        <PropertyGroup>
+            <Last>%(TGZFiles.Filename)</Last>
+        </PropertyGroup>
+
+        <!--Extracts the file to /package-->
+        <Exec Command="tar -xzf $(MSBuildProjectDirectory)\$(Last).tgz --strip-components=1 --directory=web/library"></Exec>
```

Comment:
are not following the same structure than for the other packages? something like Packages/LibrarieJS
![image](https://github.com/DynamoDS/Dynamo/assets/61755417/0482cf56-b6fd-4a07-aab2-8cd81cfb37cf)